_data/ome2021.json

```diff
@@ -13,79 +13,94 @@
   "organizer": [
     {
       "@type": "Person",
-      "name": "Sebastien Besson"
+      "name": "Sébastien Besson",
+      "affiliation": {
+        "@type": "Organization",
+        "name": "University of Dundee"
+      }
     },
     {
       "@type": "Person",
-      "name": "Jean-Marie Burel"
+      "name": "Jean-Marie Burel",
+      "affiliation": {
+        "@type": "Organization",
+        "name": "University of Dundee"
+      }
     },
     {
-      "@type": "Person",
-      "name": "June Matthew"
+        "@type": "Person",
+        "name": "June Matthew",
+        "affiliation": {
+          "@type": "Organization",
+          "name": "University of Dundee"
+        }
     },
     {
       "@type": "Person",
-      "name": "Josh Moore"
+      "name": "Josh Moore",
+      "affiliation": {
+        "@type": "Organization",
+        "name": "University of Dundee"
+      }
     }
   ],
   "subEvent": [
     {
       "@type": "Event",
-      "name": "Day 1",
-      "alternateName": "Overview",
+      "name": "Introduction, Keynote & Demos",
+      "alternateName": "Day 1",
       "url": "/events/ome-community-meeting-2021/day1/",
       "eventAttendanceMode": "https://schema.org/OnlineEventAttendanceMode",
-      "description": "Welcome, Keynote and Demos",
       "startDate": "2021-06-08T10:00+01:00",
       "endDate": "2021-06-08T20:00+01:00",
       "location": {
         "@type": "VirtualLocation",
         "url": "TBD"
       },
+      "performer": [
+        {
+          "@type": "Person",
+          "name": "Helen Berman",
+          "affiliation": {
+            "@type": "Organization",
+            "name": "RCSB PDB"
+          },
+          "sameAs": "https://www.rcsb.org/pages/about-us/helen-berman"
+        }
+      ],
       "subEvent": [
         {
           "@type": "Event",
-          "name": "TBD",
+          "name": "Introduction",
           "startDate": "2021-06-08T10:00+01:00",
           "endDate": "2021-06-08T11:00+01:00",
-          "description": "Keynote",
-          "performer": {
-            "@type": "Person",
-            "name": "TBD",
-            "affiliation": {
-              "@type": "Organization",
-              "name": "TBD"
-            }
-          },
-      "location": {
-        "@type": "VirtualLocation",
-        "url": "TBD"
-      }
+          "location": {
+            "@type": "VirtualLocation",
+            "url": "TBD"
+          }
         },
         {
           "@type": "Event",
           "name": "Demos",
-          "description": "Keynote",
           "startDate": "2021-06-08T11:00+01:00",
           "endDate": "2021-06-08T14:00+01:00",
-      "location": {
-        "@type": "VirtualLocation",
-        "url": "TBD"
-      }
+          "location": {
+            "@type": "VirtualLocation",
+            "url": "TBD"
+          }
         },
         {
           "@type": "Event",
-          "name": "TBD",
+          "name": "Keynote",
```

Comment:
Keynote speaker ?
Or maybe
Keynote lecture ? 
In combination with a concrete name below, the fuller title would sound better imho.

Reply:
I think the `Keynote speaker` is the `perfomer` of the `Event`. For the name, `Keynote Lecture` makes sense. Should I add `TBD` or `Title TBD` as the description for now?

Reply:
> Should I add TBD or Title TBD as the description for now?

Yes, Title TBD would, I think, make it even more solid.